src/node/rpc/call_types.h

```diff
@@ -139,12 +139,43 @@ namespace ccf
   {
     struct In
     {
-      int64_t commit = 0;
+      std::string transaction_id;
+    };
+
+    struct Node
+    {
+      std::optional<std::string> left = std::nullopt;
+      std::optional<std::string> right = std::nullopt;
     };
 
     struct Out
     {
-      std::vector<std::uint8_t> receipt = {};
+      std::string signature;
+      std::string root;
+      std::vector<Node> proof = {};
```

Comment:
Think this type should be something other than `Node`, since its nothing to do with the `ccf::NodeId node_id` just below. Maybe `MerkleStep` or `ProofElement` or `Link` or similar?

Reply:
It's a node in the MerkleTree though and a GetReceipt::Node, not a ccf::Node. If we had an accurate alternative I would agree, but I don't feel like the other terms proposed are more precise.

Reply:
`MerkleNode`?

Reply:
Path step, proof step, path element, tree node. Between two nodes there's also a "branch"?

Reply:
I was quite tempted by `NeighbourNode` but there's the whole British/American thing, so maybe `Element`. It still feels like a failure to namespace more than a legitimate change.